prow/cmd/deck/job_history.go

```diff
@@ -302,13 +302,16 @@ func getBuildData(bucket storageBucket, dir string) (buildData, error) {
 	finished := gcs.Finished{}
 	err = readJSON(bucket, path.Join(dir, "finished.json"), &finished)
 	if err != nil {
+		b.Result = "Pending"
 		logrus.Infof("failed to read finished.json (job might be unfinished): %v", err)
```

Comment:
nit: could we move to a debug? or remove the line?

Reply:
debugified